frontend/src/constants/Routes.ts

```diff
@@ -16,6 +16,8 @@ export const ASSESSMENT_EDITOR_PAGE = "/admin/assessment-editor";
 export const TEACHER_LANDING_PAGE = "/teacher";
 export const TEACHER_DASHBOARD_PAGE = "/teacher/dashboard";
 export const DISPLAY_ASSESSMENTS_PAGE = "/teacher/assessments";
+export const DISPLAY_ASSESSMENTS_BY_STATUS_PAGE = (status: string) =>
+  `${DISPLAY_ASSESSMENTS_PAGE}/${status}`;
```

Comment:
```suggestion
  `${DISPLAY_ASSESSMENTS_PAGE}/status=${status}`;
```
would this look better? idk

Reply:
Assuming you meant ``` `${DISPLAY_ASSESSMENTS_PAGE}?status=${status}` ```? If so, the problem then is that we have existing machinery to handle tabs-with-routes that I'd prefer not to refactor, and that code depends on the parameter being a URL parameter (accessible via `useParams`) rather than in the query string.